pkg/gen/input/dependabot/internal/file/dependabot.go

```diff
@@ -0,0 +1,48 @@
+package file
+
+import (
+	"path/filepath"
+
+	"github.com/giantswarm/devctl/pkg/gen/input"
+	"github.com/giantswarm/devctl/pkg/gen/input/dependabot/internal/params"
+)
+
+func NewCreateDependabotInput(p params.Params) input.Input {
+	i := input.Input{
+		Path:         filepath.Join(p.Dir, "dependabot.yml"),
+		TemplateBody: createDependabotTemplate,
+		TemplateDelims: input.InputTemplateDelims{
+			Left:  "{{",
+			Right: "}}",
+		},
+		TemplateData: map[string]interface{}{
+			"Reviewers": p.Reviewers,
+		},
+	}
+
+	return i
+}
+
+var createDependabotTemplate = `# DO NOT EDIT. Generated with:
+#
+#    devctl gen dependabot
+#
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
This should be configurable IMO

Reply:
Why?

Reply:
Some repositories e.g. provider operators are getting updated quite often. So I would like to have daily for them.

Reply:
Why not daily for everything then?

Reply:
Because some libraries are not so important - I would not want to have daily updates to `k8s-kvm` or something :D

Reply:
I'm not convinced. Weekly seems to be good enough for me. But I won't block if that's what you want.

Reply:
Just my 2 cents:
Besides better velocity, daily actually provides a better dev experience because the PRs come one at a time, so they get integrated and tested individually. The overall time spent updating is shorter because the dev just approves one PR and is done for the day.

If using weekly, multiple PRs are opened at once which then need to be merged, but after merging each one the others then need time to rebase. Then there are a chunk of dependency changes. If one of them broke something, it is less clear which update it was.

If there are no updates on a given day, then there are no PRs anyway.

I'd say daily should be the default, but I'm fine configuring it.

Reply:
I'm not convinced we should use different value in some places. Either daily or weekly works for me.

Reply:
I added a flag `--daily` to overwrite config from weekly schedule.